### DISCUSSION TITLE
Mountain Duck Windows 11 Taskbar Icon Appears Black Making It Difficult To See

### DISCUSSION THREAD

**Discussion by WhatImListnTo** [iterate-ch/cyberduck#15563] [Q&A]
I bought a new Windows 11 computer upgrading from Windows 10 on my old machine.  The taskbar icon for Mountain Duck was white under Windows 10.  Under Windows 11 it is black.  The background is dark, and the black icon is very difficult to see.  The icon remains black whether I have established an FTP connection or not.  Note that Cyberduck does not have a taskbar icon, so this issue does not apply to it.  Mountain Duck does include one.  

This is a minor issue.  All functionality with Mountain Duck is working as expected under Windows 11.

**Comment by AliveDevil** [comment]
Please let me know which options are set in
```
Windows Settings -> Personalization -> Colors.
```
[Change Colors](https://support.microsoft.com/windows/d26ef4d6-819a-581c-1581-493cfcc005fe)

**Comment by WhatImListnTo** [comment]
Choose your mode is set to: Light.  It appears this is the reason that the taskbar icon appears as black.  When I change it to Dark the taskbar icon appears as white.  I prefer Light.  The issue is that my taskbar is dark blue that results in the icon being difficult to see.

**Reply by AliveDevil** [reply]
I’ll check, and see what we can do to improve the situation here.
Black on dark definitely shouldn’t happen.

**Comment by WhatImListnTo** [comment]
The most likely reason this is happening to me is that I do not like Windows 11 default colors.  If I had set it to light and left my taskbar color as light, this most likely would not be an issue.  I just do not like Microsoft's default colors under Windows 11.  I don't like a lot of changes MS decided to make, unfortunately.

**Reply by AliveDevil** [reply]
Can you let me know which options are set exactly in order to reproduce the wrong icon choice?

**Reply by WhatImListnTo** [reply]
I will share a screenshot of Windows 11 Settings within the next few hours.

The easiest way to solve this issue is to stop using transparency in your taskbar icons.  Give them all backgrounds that make the icons clearly seen.  By doing this you would not need two different sets of icons for light and dark.

An alternative solution to above is to add a border around the icon to make it more visible.  This would require multiple icons, because the border must negate the opposing background color.

**Comment by WhatImListnTo** [comment]
Here is a screenshot of my WIndows 11 Settings for Personalization, Colors.  My taskbar color is currently set to match what is shown for Windows colors that is checked off.

![Win11_Settings_Personalization_Colors](https://github.com/iterate-ch/cyberduck/assets/55523642/5e1b7ef0-ef0d-498d-ad8a-c144ccf9fefe)

**Reply by AliveDevil** [reply]
To confirm: You did set `ColorPrevalence` to 1 in `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize`?

**Reply by WhatImListnTo** [reply]
Negative.  ColorPrevalence is currently set to zero in my registry at that location.

**Comment by WhatImListnTo** [comment]
Here are two screenshots of my taskbar under Windows 11.  The first is with no connection established.  The second is with a connection established.

![Window11_Taskbar_Icons_Not_Connected](https://github.com/iterate-ch/cyberduck/assets/55523642/a8fdd6c1-89b5-4b73-a655-79908b5ccbd6)

![Window11_Taskbar_Icons_Connected](https://github.com/iterate-ch/cyberduck/assets/55523642/d9bb9123-8594-420b-a48b-de4d64c874f9)
